kythe/cxx/indexer/cxx/IndexerASTHooks.cc

```diff
@@ -2018,6 +2018,22 @@ bool IndexerASTVisitor::VisitDeclRefOrIvarRefExpr(
     auto StmtId = BuildNodeIdForImplicitStmt(Expr);
     if (auto RCC = RangeInCurrentContext(StmtId, Range)) {
       GraphObserver::NodeId DeclId = BuildNodeIdForRefToDecl(FoundDecl);
+      if (ShouldHaveBlameContext(FoundDecl)) {
+        if (Job->BlameStack.empty()) {
+          if (auto FileId = Observer.recordFileInitializer(RCC.value())) {
+            Observer.recordBlameLocation(
+                RCC.value(), FileId.value(),
```

Comment:
nit: prefer unary-operator* over `.value()` when the instance has already been verified to contain a value (as these have been, and below).